app/controllers/profile/census.js

```diff
@@ -17,12 +17,13 @@ const {
 } = decennial;
 
 export default Controller.extend({
-  profile: controller('profile'),
+  // arguments
+  model: {},
 
-  data: computed('model.taskInstance.value', function() {
-    return this.get('model.taskInstance.value');
-  }),
```

Comment:
~Hmm does this still work? This seems important...~

Actually, it doesn't appear that census.hbs even references a property called `data` lol.